features/google/src/views/explore/explore.jsx

```diff
@@ -37,11 +45,44 @@ const ReportCard = () => {
 
 const ExploreView = () => {
     const { account } = useContext(PolyImportContext);
-    const { reportIsSent } = useContext(GoogleContext);
+    const { reportIsSent, handleReportSent } = useContext(GoogleContext);
 
     const history = useHistory();
     const exploreRef = useRef();
 
+    const handleCloseReportResult = () => {
+        handleReportSent(null);
+    };
+
+    const renderReportResult = () =>
+        reportIsSent !== null && (
+            <PopUpMessage
+                reportResultAnswer={
+                    reportIsSent ? " successfully" : " unsuccessfully"
```

Comment:
No translation here? The user is not getting this message, right?

Reply:
You can see the translations above (line 66 and 75). `"successfully"` and  `"unsuccessfully"` are just the classNames to change the styles depending on the message.